tests/test_data_generators.py

```diff
@@ -1,26 +1,29 @@
-import os
 import json
+import os
+from pathlib import Path
+
 import numpy as np
 import pytest
-from matchms import Spectrum
+
 from ms2deepscore import BinnedSpectrum
 from ms2deepscore.data_generators import DataGeneratorAllInchikeys
 from ms2deepscore.data_generators import DataGeneratorAllSpectrums
-path_tests  = os.path.dirname(__file__)
+
+TEST_RESOURCES_PATH = Path(__file__).parent / 'resources'
```

Comment:
I think the API of pathlib is much cleaner than `os.path`, but I am curious what you think?

Reply:
I agree. Lourens recently suggested this to me as well for matchms (I made an issue there and hope implement it there as well soonish). So, yes, totally up for it!